index.md

```diff
@@ -89,14 +78,14 @@ ms.assetid:
                                         <div class="card">
                                             <div class="cardImageOuter">
                                                 <div class="cardImage">
-                                                    <img src="./docs/images/hub/net-gs-1.svg" alt="" />
+                                                    <img src="articles/images/hub/logo_NET.svg" alt="" />
                                                 </div>
                                             </div>
                                             <div class="cardText">
                                                 <h3><a href="/dotnet/articles/welcome">What is .NET?</a></h3>
-                                                <p><a href="/dotnet/articles/standard/getting-started">Get Started</a></p>
-                                                <p><a href="/dotnet/articles/standard/tour">Tour of .NET</a></p>
-                                                <p><a href="/dotnet/articles/standard/components">.NET Architectural Concepts</a></p>
+                                                <p><a href="https://www.microsoft.com/net">Get Started</a></p>
+                                                <p><a href="/dotnet/articles/standard/#a-stroll-through-net">Tour of .NET</a></p>
+                                                <p><a href="/dotnet/articles/about/products">.NET Concepts</a></p>
```

Comment:
I'll have to check these. It's likely that the version in /dotnet/docs was updated separately from the version in the production team's sandbox. I can revert these.